packages/actions/src/Imports/Events/ImportCsvProcessed.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Filament\Actions\Imports\Events;
+
+use Filament\Actions\Imports\Models\Import;
+use Illuminate\Foundation\Events\Dispatchable;
+use Throwable;
+
+class ImportCsvProcessed
+{
+    use Dispatchable;
+
+    /**
+     * @param array<Throwable>  $exceptions
+     */
+    public function __construct(
+        protected Import $import,
+        protected int $processedRows,
+        protected int $successfulRows,
```

Comment:
Can you please confirm why you are passing these here since they are already available in the `$import` object?

Reply:
These numbers pertain only to the current job. However, the `$import` object contains cumulative numbers for all processed jobs.